lib/pubid/core/identifier/base.rb

```diff
@@ -74,6 +84,48 @@ def to_s
         self.class.get_renderer_class.new(get_params).render
       end
 
+      def typed_stage_abbrev
+        if self.class::TYPED_STAGES.key?(typed_stage)
+          self.class::TYPED_STAGES[typed_stage][:abbr]
```

Comment:
```ruby
if ...
  return self.class::TYPED_STAGES[typed_stage][:abbr]
end
```

Reply:
@ronaldtse why do we need return here?

Reply:
Because there's no other condition after this. It's just a "shortcut pattern":

```ruby
def func(x)
  if x > 100
    "a"
  else
    "b"
  end
end
```

Can be more succinctly written as:
```ruby
def func(x)
  if x > 100
    return "a"
  end

  "b"
end
```

That's it.